Trimble.Modus.Components/Helpers/ThemeManager.cs

```diff
@@ -12,21 +12,53 @@ public class ThemeManager
 
         public static void Initialize(ModusConfig appConfig = null)
         {
-            var defaultLightThemeColors = new Styles.LightThemeColors();
-            var defaultDarkThemeColors = new Styles.DarkThemeColors();
-
             IncludeFromResources(new Styles.Colors());
-      
-            LightThemeColorResourceDictionary = UpdateDictionary(appConfig.LightThemeColors, defaultLightThemeColors);
-            DarkThemeColorResourceDictionary = UpdateDictionary(appConfig.DarkThemeColors, defaultDarkThemeColors);
-            LightThemeStylingResourceDictionary = appConfig.LightThemeStyles;
-            DarkThemeStylingResourceDictionary = appConfig.DarkThemeStyles;
-
+            UpdateColorsDictionary(appConfig);
+            UpdateStylingDictionary(appConfig);
             UpdateTheme(Application.Current.RequestedTheme);
 
             Application.Current.RequestedThemeChanged += Current_RequestedThemeChanged;
         }
 
+        private static void UpdateColorsDictionary(ModusConfig appConfig = null)
+        {
+            var defaultLightThemeColors = new Styles.LightThemeColors();
+            var defaultDarkThemeColors = new Styles.DarkThemeColors();
+            if (appConfig != null)
```

Comment:
Thanks for fixing this. We faced this issue in our app as well. We had to pass a blank AppConfig